tools/src/merger/OpenApiMerger.ts

```diff
@@ -40,17 +45,27 @@ export default class OpenApiMerger {
     }
   }
 
-  merge (output_path: string = ''): OpenAPIV3.Document {
+  write_to(output_path: string): OpenApiMerger {
+    if (!this._merged) { throw "Call merge() first." }
```

Comment:
Why doesn't `write_to()` just call `merge()` itself in this case?

Reply:
I got rid of `merge()` altogether, it just does it the first time when you call `spec()`.